common/vm_get_devices_with_ctrl_key.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get VM device with the device key
```

Comment:
I think this description can be more precise about devices with the same controller.